Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/DefaultModelInfoProvider.java

```diff
@@ -52,10 +52,8 @@ public boolean accept(File path, String name) {
                     String[] fileNameComponents = fileName.split("-");
                     if (fileNameComponents.length == 3) {
                         Version version = new Version(fileNameComponents[2]);
-                        if (requestedVersion == null ||
-                                (version.isComparable(requestedVersion) && version.compatibleWith(requestedVersion))) {
-                            if (mostRecent == null ||
-                                    (version.isComparable(requestedVersion) && version.compareTo(mostRecent) > 0)) {
+                        if (requestedVersion == null || version.compatibleWith(requestedVersion)) {
+                            if (mostRecent == null || version.compareTo(mostRecent) > 0) {
```

Comment:
Same comment applies here, the compareTo method can only be used on comparable versions, otherwise strict equality.

Reply:
Updated this. Thanks!

Reply:
Same comment here, needs to use equality if the versions aren't comparable